### PR TITLE
Submit Allspark Adapter

### PR DESCRIPTION
**Pull request by MaxwellBlock**
**NOTE**

### Please enable "Allow edits by maintainers" while putting up the PR.
____

## Type of change
- [x] New adapters
- [ ] Adapter-related bug fix

Description:

___

### Description

Please provide the following information based on the type of points calculation:

#### For TVL Points Calculation:

1. Dapp Description:
   

2. Criteria for Calculating User Points:
   

3. Collateral Asset Locking Contract:
   
   > **Note**: If there are multiple `tokenAddress`, please specify which contracts the different `tokenAddress` are locked in.

5. LP Contract:
   
   > **Note**: User balance is calculated as LP percentage multiplied by total collateral.

#### For Volume/Transaction Points Calculation:

1. Dapp Description:
   trade game.

2. User Points Calculation Criteria:
   0.5 points per transaction.

3. Solidity Contract Signature Involved in Transactions:
   OrderPlaced(bool isGreater, address guesser, uint256 nonce, uint256 amount, uint256 guessPrice);

4. Conditions for User Points Calculation:
   Users triggering the OrderPlaced event in the 0xD06B5A208b736656A8F9cD04ed43744C738BD8A9 contract will earn 0.5 points.

5. Quantity Calculation Rule:
   it is the value of `amount` (ETH).

### CODE REVIEW

**Comment by xsteadybcgo** [comment]
I get all the price is **15**, i think it is not an incorrect price for ETH.

File: src/adapters/allspark/execution/src/sdk/lib.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  const data = await response.json();
+  const userTxData: UserTxData[] = data.data.map((item: any) => {
+    return {
+      timestamp: item.timestamp,
+      userAddress: item.userAddress,
+      contractAddress: item.contractAddress,
+      tokenAddress: item.tokenAddress,
+      decimals: item.decimals,
+      price: item.price,
```
Comment (xsteadybcgo) [inline src/adapters/allspark/execution/src/sdk/lib.ts:18]: you can omit the price and remove this field

File: src/adapters/allspark/adapters.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+3. Solidity Contract Signature Involved in Transactions:
+   OrderPlaced(bool isGreater, address guesser, uint256 nonce, uint256 amount, uint256 guessPrice);
+
+4. Conditions for User Points Calculation:
+   Users triggering the Swap event in the 0xD06B5A208b736656A8F9cD04ed43744C738BD8A9 contract will earn 0.5 points.
+
+5. Quantity Calculation Rule:
+    it is the value of `amount`
+```
```
Comment (xsteadybcgo) [inline src/adapters/allspark/adapters.md:35]: This file can be removed

File: src/adapters/allspark/execution/src/sdk/lib.ts (hunk 1 above)
Comment (MaxwellBlock) [reply src/adapters/allspark/execution/src/sdk/lib.ts]: solved

File: src/adapters/allspark/adapters.md (hunk 2 above)
Comment (MaxwellBlock) [reply src/adapters/allspark/adapters.md]: solved

**Pull request merged**